MediaBrowser.Common/Net/MoreHeaderNames.cs

```diff
@@ -0,0 +1,83 @@
+using HN = Microsoft.Net.Http.Headers.HeaderNames;
+
+namespace MediaBrowser.Common.Net
+{
+    public static class MoreHeaderNames
+    {
+        // Other Headers
+        public const string XForwardedFor = "X-Forwarded-For";
+        public const string XForwardedPort = "X-Forwarded-Port";
+        public const string XForwardedProto = "X-Forwarded-Proto";
+        public const string XRealIP = "X-Real-IP";
+
+        // Headers from Microsoft.Net.Http.Headers.HeaderNames
```

Comment:
I'm not a fan of this. I'd rather we just have a separate `CustomHeaderNames` that supplemented MS' HeaderNames.